packages/owl-bot/src/create-pr.ts

```diff
@@ -15,6 +15,23 @@
 import {newCmd} from './cmd';
 import {OctokitType} from './octokit-util';
 
+/***
+ * Github will reject the pull request if the title is longer than 255
+ * characters.  This function will move characters from the title to the body
+ * if the title is too long.
+ */
+export function resplit(
+  title: string,
+  body: string
```

Comment:
FYI I guess the pull request body also has a limit of 65535. Probably you're fine.

Reply:
Thanks @tmatsuo.  I'll add that limit too.  Seems like we always get bitten eventually.